toot/tui/overlays.py

```diff
@@ -30,17 +30,27 @@ def __init__(self, status_details):
 
 class StatusLinks(urwid.ListBox):
     """Shows status links."""
+    signals = ["clear-screen"]
 
     def __init__(self, links):
 
         def widget(url, title):
-            return Button(title or url, on_press=lambda btn: webbrowser.open(url))
+            return Button(title or url, on_press=lambda btn: self.browse(url))
 
         walker = urwid.SimpleFocusListWalker(
             [widget(url, title) for url, title in links]
         )
         super().__init__(walker)
 
+    def browse(self, url):
+        webbrowser.open(url)
+        wb = webbrowser.get()
+        if wb.name == 'www-browser':
+            # 'www-browser' implies a generic
+            # console browser; once it exits
+            # we need to refresh the tui screen
+            self._emit("clear-screen")
```

Comment:
In my testing, I don't get `www-browser` when I configure  elinks as browser. 

```
λ BROWSER=elinks python                                                     
Python 3.10.8 (main, Nov 14 2022, 00:00:00) [GCC 12.2.1 20220819 (Red Hat 12.2.1-2)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import webbrowser
>>> webbrowser.get().name
'elinks'
```

I'm guessing you're using the [www-browser ubuntu package](https://packages.ubuntu.com/bionic/www-browser). 

Does it hurt if we clear the screen after exiting any browser, even if it's graphical?

Reply:
Correct, I am using www-browser in ubuntu.  I switched to Firefox for testing purposes and I can confirm that refreshing the screen after exiting any browser has no negative effect. I didn't even see a screen flicker in the console.  So I'll remove the check and execute the refresh every time.

Reply:
Fixed